windows/src/desktop/kmshell/xml/splash.css

```diff
@@ -66,7 +62,7 @@ div {
   max-height: 84px;
   overflow-y: auto;
   width: 400px;
-  padding: 0 16px 0 0;
+  padding: 1px 16px 0 0;
```

Comment:
Can you explain why you changed the padding for the splash #keyboards element?

Reply:
In windows 10, Kannada keyboard name is cut off at the top. All Indic languages, also I presume Arabic uses double line width. It could be the reason. 
But I will try to reduce the number to 0.1px to see if it's enough to fix the issue.

Reply:
padding is removed now